<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>文件管理器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/folder.ico</normaloff>:/icons/folder.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
        <property name="gridSize">
         <size>
          <width>200</width>
          <height>80</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
        <property name="uniformItemSizes">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_list"/>
   <addaction name="action_icon"/>
   <addaction name="action_refresh"/>
   <addaction name="action_higher"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_icon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="test" resource="icons.qrc">
     <normaloff>:/icons/iocn.ico</normaloff>:/icons/iocn.ico</iconset>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
   <property name="toolTip">
    <string>按图标方式查看</string>
   </property>
  </action>
  <action name="action_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/list.ico</normaloff>:/icons/list.ico</iconset>
   </property>
   <property name="text">
    <string>list</string>
   </property>
   <property name="toolTip">
    <string>按列表方式查看</string>
   </property>
  </action>
  <action name="action_higher">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/up.ico</normaloff>:/icons/up.ico</iconset>
   </property>
   <property name="text">
    <string>higher</string>
   </property>
   <property name="toolTip">
    <string>返回上一级目录</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/refresh.ico</normaloff>:/icons/refresh.ico</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
